Snowflake.Data.Tests/UnitTests/Session/SFHttpClientPropertiesTest.cs

```diff
@@ -45,7 +45,7 @@ public class SFHttpClientPropertiesTest
             var parameterMap = properties.ToParameterMap();
             
             // assert
-            Assert.AreEqual(2, parameterMap.Count);
+            Assert.AreEqual(3, parameterMap.Count);
```

Comment:
you should also test existence of the new parameter in the map

Reply:
Done